### PR TITLE
keystone 0.9.1 (new formula)

### PR DESCRIPTION
**Pull request by stephengroat**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
add keystone

### CODE REVIEW

File: Formula/keystone.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Keystone < Formula
+  desc "Keystone assembler framework: Core (Arm, Arm64, Hexagon, Mips, PowerPC, Sparc, SystemZ & X86) + bindings"
```
Comment (ilovezfs) [inline Formula/keystone.rb]: Please make this briefer.

Comment (stephengroat) [reply Formula/keystone.rb]: done

**Comment by stephengroat** [comment]
should be good and ready for review

**Comment by woodruffw** [comment]
This LGTM; any other thoughts here @ilovezfs?

**Review by ilovezfs — APPROVED** [review]

**Comment by woodruffw** [comment]
@BrewTestBot test this please

**Comment by woodruffw** [comment]
@stephengroat one small audit failure:

```
  * C: 2: col 9: Description shouldn't start with the formula name
```

**Comment by stephengroat** [comment]
@woodruffw sorry about that, fixed

**Comment by woodruffw** [comment]
Thanks @stephengroat!

**Pull request closed without merging**